addons/event_sale/wizard/event_edit_registration.py

```diff
@@ -9,6 +9,7 @@ class RegistrationEditor(models.TransientModel):
 
     sale_order_id = fields.Many2one('sale.order', 'Sales Order', required=True)
     event_registration_ids = fields.One2many('registration.editor.line', 'editor_id', string='Registrations to Edit')
+    show_event_column = fields.Boolean(help="Technical field used to determine if the event column must be shown or not")
```

Comment:
Mmh could we make this a regular computed field?
I think it will work with the default_get implementation because it's a wizard but it's still a little strange.

Reply:
Actually yes, we can use a computed field for this one.